core/provider/service.go

```diff
@@ -260,24 +260,30 @@ func (s *Service) FetchResources(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-
 	failedProviders := map[string]error{}
+	resourcesCount := 0
+	upsertedResourceCount := 0
 	for _, p := range providers {
 		startTime := time.Now()
 		s.logger.Info(ctx, "fetching resources", "provider_urn", p.URN)
-		resources, err := s.getResources(ctx, p)
+		resources, err := s.getResources(ctx, p, &resourcesCount, &upsertedResourceCount)
```

Comment:
suggesting to rename the function to `fetchNewResources` or similar, as it is now no longer returning the complete list of resources (at least one of the reason)